components/Front/index.js

```diff
@@ -78,7 +116,33 @@ export default compose(
       }
 
       return {
-        data
+        data,
+        fetchMore: () => {
+          return data.fetchMore({
+            variables: {
+              first: 5,
+              after: data.front.children.pageInfo.endCursor
+            },
+            updateQuery: (previousResult = {}, { fetchMoreResult = {} }) => {
+              const previousSearch = previousResult.front.children || {}
+              const currentSearch = fetchMoreResult.front.children || {}
+              const previousNodes = previousSearch.nodes || []
+              const currentNodes = currentSearch.nodes || []
+              const res = {
```

Comment:
Maybe `return` here already.

Reply:
Yeah, thats remains of a log call. Will do. Thanks!